tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,340 @@
+        mt.GT,
+        aaf_threshold=aaf_threshold,
+        male_threshold=xy_fstat_threshold,
+        female_threshold=xx_fstat_threshold,
+    )
+    mt = mt.annotate_cols(**sex_ht[mt.col_key])
+    sex_ht = mt.cols()
+
+    # Plot histogram of fstat values
+    df = sex_ht.to_pandas()
+    plt.clf()
+    plt.hist(df["f_stat"])
+    plt.xlabel("Fstat")
+    plt.ylabel("Frequency")
+    plt.axvline(xy_fstat_threshold, color="blue", linestyle="dashed", linewidth=1)
+    plt.axvline(xx_fstat_threshold, color="red", linestyle="dashed", linewidth=1)
+
+    outplot = f"{outdir}/fstat_{callset_name}.png"
+    with hl.hadoop_open(outplot, "wb") as out:
+        plt.savefig(out)
+
+    return sex_ht
+
+
+def call_sex(
+    callset: str,
+    use_y_cov: bool = False,
+    y_cov_threshold: float = 0.1,
+    normalization_contig: str = "20",
+    xy_fstat_threshold: float = 0.75,
+    xx_fstat_threshold: float = 0.5,
+    aaf_threshold: float = 0.05,
+    call_rate_threshold: float = 0.25,
+) -> hl.Table:
+    """
+    Call sex for the samples in a given callset and export results file to the callset directory.
+
+    :param str callset: String of full MatrixTable path for the callset
+    :param use_y_cov: Set to True to calculate and use chrY coverage for sex inference. Default is False
+    :param y_cov_threshold: Coverage on chrY above which supports male call. Default is 0.1
+    :param normalization_contig: Chosen chromosome for calculating normalized coverage. Default is "20"
+    :param xy_fstat_threshold: F-stat threshold above which a sample will be called XY. Default is 0.75
+    :param xx_fstat_threshold: F-stat threshold below which a sample will be called XX. Default is 0.5
+    :param aaf_threshold: Alternate allele frequency threshold for `hl.impute_sex`. Default is 0.05
+    :param call_rate_threshold: Minimum required call rate. Default is 0.25
+    :return: Table with sex annotations
+    """
+
+    # Read in matrix table and define output directory
+    # TODO: Generalize before moving into gnomad_methods
+    outdir = dirname(callset)
+    mt_name = callset.split("/")[-1].strip("\.mt")
+    logger.info("Reading matrix table for callset: %s", callset)
+    logger.info("Using chromosome Y coverage? %s", use_y_cov)
+
+    mt = hl.read_matrix_table(callset)
+
+    # Filter to SNVs and biallelics
+    mt = mt.filter_rows(
+        (hl.len(mt.alleles) == 2) & hl.is_snp(mt.alleles[0], mt.alleles[1])
+    )
+
+    # Filter to pass variants only (empty set)
+    # TODO: Make this an optional argument before moving to gnomad_methods
+    mt = mt.filter_rows(mt.filters.length() == 0, keep=True)
+
+    # Infer build:
+    build = get_reference_genome(mt.locus).name
+    logger.info("Build inferred as %s", build)
+
+    logger.info("Inferring sex...")
+    # TODO: Change "female" and "male" to "XX" and "XY"
+    if use_y_cov:
+        sex_ht = get_chr_cov(mt, "GRCh38", normalization_contig)
+        mt = mt.annotate_cols(**sex_ht[mt.col_key])
```

Comment:
```suggestion
        norm_ht = get_chr_cov(mt, "GRCh38", normalization_contig)
        mt = mt.annotate_cols(**norm_ht[mt.col_key])
```
(making the HT name more descriptive)